### ISSUE TITLE
Reordering list items by dragging them

### ISSUE THREAD

**Issue by DNin01** [ScratchAddons/ScratchAddons#6735]
### Why this would be helpful

Sometimes it can be helpful to be able to rearrange items within a list, like if you have a hardcoded list of, I don't know, pictures to display in a certain order. You can see how it might be tedious editing that list when you want to keep rearranging things or inserting new items in-between others.

### How this addon works

I want to see an addon that allows you to drag items within a list monitor to reorder them. It could also optionally support inserting items between others, maybe by clicking the space between two list items.

**Comment by WorldLanguages** [comment]
Could be part of "variable manager"

**Comment by WorldLanguages** [comment]
Yeah, adding drag-and-drop to the list monitor itself sounds like quite a challenge. Scratch uses a "virtual list" React component for it, which leads to numerous problems.

**Comment by BroJac5246** [comment]
> Could be part of "variable manager"

Good idea! I would love to see this in a new addon as well, though, because I personally find the variable manager a little excessive for what I would need it for.

**Comment by DNin01** [comment]
> Yeah, adding drag-and-drop to the list monitor itself sounds like quite a challenge. Scratch uses a "virtual list" React component for it, which leads to numerous problems.

I understand. Seems like a reasonable first step.
